### DISCUSSION TITLE
Testing with vite and vitest

### DISCUSSION THREAD

**Discussion by akapulka** [quasarframework/quasar#13791] [General - Components / Directives / etc] [answered]
As there are no docs yet how do I test with vite and vitest? 
Everything works execpt component rendering.
How can I render quasar components while testing?

**Comment by IlCallo** [comment, marked as answer]
Check out the newly released Vitest App Extension: https://github.com/quasarframework/quasar-testing/releases/tag/unit-vitest-v0.1.1

**Reply by akapulka** [reply]
Yes exactly what I needed, thanks!
